### DISCUSSION TITLE
Menus and pages

### DISCUSSION THREAD

**Discussion by marcelloinfoweb** [pratikborsadiya/vali-admin#136] [Q&A]
When I create the pages and link in the new menu, it is neither expanded nor active.

How can I edit pages, rename menus, and rename pages to php?

How is it possible?

Thank you...

**Comment by micksp** [comment]
I ran into the same problem and couldn't find any code that should have done the menu expanding. I added this to the javascript in main.js, just above the comment: `// Activate sidebar treeview toggle`: 
```javascript
// set correct treeview to is-expanded, depending on the URL
if(path !== '/') {
	// find the entry that contains the url as href
	var $item = $(".treeview-item[href='" + url + "']")
		.parents(".treeview").addClass("is-expanded")
}
```
That did the job for me. I will now see if can find the time to see what this "pug" is all about, so I can make a pull request to add this.

**Comment by vishaalmandge** [comment]
@micksp Its showing an error path is not defined
